packages/lesswrong/components/tagging/AllTagsPage.tsx

```diff
@@ -45,6 +45,17 @@ const styles = (theme: ThemeType): JssStyles => ({
   addTagButton: {
     verticalAlign: "middle",
   },
+  coreTagsSection: {
+    /**
+     * I'm slightly abusing the ToCColumn component by using an empty table of contents for the top core tags section.
+     * This makes it way easier to line up with the rest of the page, while keeping the table of contents for the
+     * lower wiki section. This works fine apart from the fact that the table of contents component has a fixed height of 80vh,
+     * this removes that
+     */
```

Comment:
This is not a docstring
```suggestion
    // I'm slightly abusing the ToCColumn component by using an empty table of contents for the top core tags section.
    // This makes it way easier to line up with the rest of the page, while keeping the table of contents for the
    // lower wiki section. This works fine apart from the fact that the table of contents component has a fixed height of 80vh,
    // this removes that
 ```
 Very good comment though.